crates/admin/src/cluster_controller/service.rs

```diff
@@ -543,17 +583,19 @@ impl<T: TransportConnect> Service<T> {
                 info!(
                     ?log_id,
                     trim_point_inclusive = ?trim_point,
-                    "Manual trim log command received");
+                    "Trim log command received");
                 let result = self.bifrost.admin().trim(log_id, trim_point).await;
                 let _ = response_tx.send(result.map_err(Into::into));
             }
             ClusterControllerCommand::CreateSnapshot {
                 partition_id,
+                min_target_lsn,
                 response_tx,
             } => {
                 info!(?partition_id, "Create snapshot command received");
-                self.create_partition_snapshot(partition_id, response_tx)
+                self.create_partition_snapshot(partition_id, min_target_lsn, response_tx)
                     .await;
+                // todo(pavel): trim the log on successful snapshot
```

Comment:
I'll do it either in this PR, or as a fast follow - basically reusing the safety checks from the auto-trim logic to make sure we don't trip up any active PPs that may be slightly behind the leader.